proxy/logging/LogBuffer.cc

```diff
@@ -112,14 +112,6 @@ LogBufferHeader::log_filename()
   return addr;
 }
 
-/*-------------------------------------------------------------------------
-  LogBuffer::LogBuffer
```

Comment:
IMO it wasn't saying anything informative or useful now that AbstractBuffer is gone. If I had to add a comment it would be about what LogBuffer does and I would prefer to do that in LogBuffer.h. 

I'm deep into the weeds of the logging system right now. Afterwards, I'll fix up and add more documentation.